SDWebImage/Core/SDImageCoderHelper.m

```diff
@@ -596,13 +595,10 @@ + (BOOL)shouldScaleDownImage:(nonnull UIImage *)image limitBytes:(NSUInteger)byt
     }
     CGFloat destTotalPixels;
     if (bytes == 0) {
-        bytes = kDestImageLimitBytes;
+        bytes = [self defaultScaleDownLimitBytes];
```

Comment:
This should be a dynamic default value allows user to change, not hardcode to 60MB.